utils/examples/generate-examples.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import os
```

Comment:
Could we drop os and instead use 
```suggestion
from pathlib import Path
```

Makes everything easier and more readable.
We don't use `with open` and don't have to go over all the files if we use `glob`.

Reply:
@ahankinson @lpugin @musicEnfanthen please resolve this suggestion

Reply:
I am inclined to leave this suggestion for later improvement. The script runs now and does what is expected of it. We will definitely be able to improve the script further later.

Reply:
referenced this in a new issue #1213 and added it to Milestone 5.1